config/win/BUILD.gn

```diff
@@ -421,7 +446,7 @@ config("default_crt") {
       configs = [ ":dynamic_crt" ]
     } else {
       # Desktop Windows: static CRT.
-      configs = [ ":static_crt" ]
+      configs = [ ":dynamic_crt" ]
```

Comment:
Comment does not match implementation. Are you sure this must be dynamic? Seems google wants that value for a reason.

Reply:
No idea why it changed for winuwp; probably a reason but I don't know what it would be; it's been a LONG time like this...